cpp/oneapi/dal/backend/interop/error_convertor.hpp

```diff
@@ -0,0 +1,271 @@
+/*******************************************************************************
+ * Copyright 2020 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#include "daal/include/services/error_handling.h"
+#include "daal/include/services/internal/status_to_error_id.h"
+#include "oneapi/dal/exception.hpp"
+
+namespace oneapi::dal::backend::interop {
+
+void status_to_exception_default(daal::services::Status s);
+
+template <class local_convertor>
+void status_to_exception(daal::services::Status s, local_convertor alg_convertor) {
+    alg_convertor(s);
+    status_to_exception_default(s);
+}
+
+void status_to_exception_default(daal::services::Status s) {
```

Comment:
```suggestion
void status_to_exception_default(const daal::services::Status& s) {
```

Reply:
Suggest to have the same name for `status_to_exception_default` and `status_to_exception`, but overload it by `StatusConverter`. If the user wants to add custom logic to status converter in the existing code, the would not need to change the function name.